modules/packages/LinearAlgebra.chpl

```diff
@@ -927,6 +927,25 @@ proc trace(A: [?D] ?eltType) {
   return trace;
 }
 
+proc kroneckerProduct(A: [?ADom] ?eltType, B: [?BDom] eltType) {
+  if ADom.rank != 2 || BDom.rank != 2 then compilerError("Rank size not 2");
+  const (rowA, colA) = A.shape;
+  const (rowB, colB) = B.shape;
+  var C = Matrix(rowA*rowB, colA * colB);
```

Comment:
This should also take an argument `eltType=eltType` to inherit the element type of the matrix arguments.

Reply:
OK, I didn't know of it, thanks